addons/mail/static/src/components/message/message.js

```diff
@@ -456,6 +456,7 @@ class Message extends Component {
      * @param {MouseEvent} ev
      */
     _onClickAuthorAvatar(ev) {
+        ev.stopPropagation();
```

Comment:
Don't ever use `stopPropagation` because that might prevent other handlers that we didn't want to prevent.

Example clicking "anywhere" to close various popup such as systray menu would fail when the "anywhere" click happen to be on this element).

We have introduced `isEventHandled` and `markEventHandled` as an alternative to this issue.

As discussed, there are way more cases to handle than just this one, clicking on name, on mention, on any other link inside message, ...

Reply:
I didn't see your message on the task, forget about it for those that you said it worked already then (I'll double check at the end).

Reply:
I'll feel better if you double check since I could easily miss something here.